src/style.css

```diff
@@ -1,3 +1,72 @@
-body {
+/* General reset */
+* {
   margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+
+  --color-blue: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+header__logo-link {
```

Comment:
it's fine you changed it here, but you didn't change name of the class in html file and your styles didn't apply, please check it once more
ps. you also forget `.` in the beginning of the selector to define that it is a class